app/api/predictions/service.py

```diff
@@ -89,73 +90,44 @@ async def get_new_prediction(cls, prompt: str, client_websocket):
-            
-            # Query the DynamoDB table with date range filter
-            response = cls().events.scan(
-                FilterExpression=boto3.dynamodb.conditions.Attr('start_ts').lte(current_timestamp) &
-                                 boto3.dynamodb.conditions.Attr('end_ts').gte(current_timestamp)
-            )
-
-            iso_date_str = current_time.isoformat()
-            # Query the DynamoDB table with date range filter
-            response = cls().events.scan(
-                FilterExpression=boto3.dynamodb.conditions.Attr('start_ts').lte(iso_date_str) &
-                                 boto3.dynamodb.conditions.Attr('end_ts').gte(iso_date_str)
-            )
-            
-            items = response.get('Items', [])
-            if items:
-                # Return the first event from the result
-                return items[0]
-            else:
-                return None
-        except ClientError as e:
-            raise Exception(e.response['Error']['Message'])
+        # Get the current date and time
+        current_time = datetime.now()
+        iso_date_str = current_time.isoformat()
+        response = await DatabaseOperations.get_daily_event(iso_date_str)
+        items = response.get('Items', [])
+        if items:
+            # Return the first event from the result
+            return items[0]
+        else:
+            return None
 
     @classmethod
     async def get_next_event(cls):
-        try:
-            # Get the current date and time
-            current_time = datetime.now()
-            iso_date_str = current_time.isoformat()
-            # Query the DynamoDB table for events that start after the current time
-            response = cls().events.scan(
-                FilterExpression=boto3.dynamodb.conditions.Attr('start_ts').gte(iso_date_str)
-            )
-
-            items = response.get('Items', [])
-
-            if items:
-                # Find the event with the earliest start_ts
-                next_event = min(items, key=lambda item: item['start_ts'])
-                start_date = next_event['start_ts']
-                team = next_event['team'].replace('_', ' VS ')
-                return {'team': team, 'start_date': start_date}
-            else:
-                return None
-        except ClientError as e:
-            raise Exception(e.response['Error']['Message'])
+        # Get the current date and time
+        current_time = datetime.now()
+        iso_date_str = current_time.isoformat()
+        response = await DatabaseOperations.get_next_event(iso_date_str)
+        items = response.get('Items', [])
+        if items:
+            # Find the event with the earliest start_ts
+            next_event = min(items, key=lambda item: item['start_ts'])
+            start_date = next_event['start_ts']
+            team = next_event['team'].replace('_', ' VS ')
+            return {'team': team, 'start_date': start_date}
```

Comment:
Same for these, we need to param this fields. We can address it later, if so, but we should add it as tech debt. Please, file a card if you won't address this on this PR.